js/card.js

```diff
@@ -0,0 +1,12 @@
+const visitingCardData= sessionStorage.getItem("visiting-card-data");
+visitingCardData = JSON.parse(visitingCardData);
+document.getElementById('first').innerHTML = visitingCardData.first_name;
+document.getElementById('last').innerHTML = visitingCardData.last_name;
+document.getElementById('title').innerHTML = visitingCardData.title ? visitingCardData.title : '';
+document.getElementById('gmail').innerHTML = visitingCardData.email;
+document.getElementById('phone-number').innerHTML = visitingCardData.phone_number;
+document.getElementById('whatsapp').href = 'https://wa.me/' + visitingCardData.phone_number;
+document.getElementById('phone-number').href = 'tel:' + visitingCardData.phone_number;
+document.getElementById('gmail').href = 'mailto:' + visitingCardData.email;
+document.getElementById('github').href = visitingCardData.github;
+document.getElementById('website').href = visitingCardData.website
```

Comment:
Do not display the element if the data is not set. Make sure that design looks good even when none of the optional fields are set.